File: Grupo 1 - Variáveis/projeto-de-extensao/exercicios/python/Consumo.py
"""
====================================
EXERCÍCIO 1 - CONSUMO DE COMBUSTÍVEL
====================================

Peça a distância percorrida e a quantidade
de combustível consumida. Calcule o consumo médio.
"""

# double (float) -> distância percorrida
distancia = float(input("Digite a distância percorrida (km): "))

# double (float) -> combustível consumido
litros = float(input("Digite os litros consumidos: "))

# double (float) -> cálculo do consumo médio
consumo = distancia / litros

print("Consumo médio: " + str(consumo) + " km/l")

File: Grupo 1 - Variáveis/projeto-de-extensao/exercicios/javascript/Consumo.js
/*
====================================
EXERCÍCIO 1 - CONSUMO DE COMBUSTÍVEL
====================================

Peça a distância percorrida e a quantidade
de combustível consumida. Calcule o consumo médio.
*/

const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("Digite a distância percorrida (km): ", (distanciaResp) => {
    readline.question("Digite os litros consumidos: ", (litrosResp) => {
        
        // double -> distância percorrida
        let distancia = parseFloat(distanciaResp);

        // double -> combustível consumido
        let litros = parseFloat(litrosResp);

        // double -> cálculo do consumo médio
        let consumo = distancia / litros;

        console.log("Consumo médio: " + consumo + " km/l");

        readline.close();
    });
});